src/main/java/convobot/utils/DateTime.java

```diff
@@ -0,0 +1,55 @@
+package utils;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+import exceptions.ConvoBotException;
+
+/**
+ * The {@code DateTime} class provides utility methods for handling date and time operations.
+ * It includes methods for converting between {@code LocalDate} objects and formatted strings,
+ * as well as handling exceptions related to date parsing.
+ */
+public class DateTime {
+
+    private static final DateTimeFormatter inputFormatter = DateTimeFormatter.ofPattern("yyyy-MM-dd");
```

Comment:
Constant name (static final) could be uppercase and use underscores instead? Like INPUT_FORMATTER.

Reply:
Accepted.